lnrpc/routerrpc/router_backend_test.go

```diff
@@ -205,6 +205,7 @@ func testQueryRoutes(t *testing.T, useMissionControl bool, useMsat bool) {
 			}
 			return route.Vertex{1}, route.Vertex{2}, nil
 		},
+		MaxTotalTimelock: 1000,
```

Comment:
Why the change here?

Reply:
Needs to be specified so the test sets `CltvLimit`, otherwise it will default to 0

Reply:
I see, could we also test that the error is returned in this test?

Reply:
Optionally setting it now to see if error is returned